tests/Kentico.Xperience.AlgoliaSearch.Tests.csproj

```diff
@@ -1,15 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net48</TargetFramework>
-    <RootNamespace>Kentico.Xperience.AlgoliaSearch.Test</RootNamespace>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
I think that project should be multi-targeted to net48 and net6.0

Reply:
Fixed in https://github.com/Kentico/xperience-algolia/pull/29/commits/3bdae246678108e671e53255db3733ce7ff53469